README.md

```diff
@@ -40,36 +42,72 @@ The `fastify.routes` Map has a key for each path any route has been registered,
 ## Example
 
 ```js
-const fastify = require('fastify')()
+const fastify = require("fastify")();
```

Comment:
remove the first example. the second example as IIFE is perfect.

Reply:
You are entitled to your opinion about code style.   :)   Avoiding any discussion of the preference of await vs then is what I went for.

Reply:
It is an example, and using `then()` is kind of strange. With the IIFE you can use async await, which is  closer to the real world case. So people can copy paste it to their code.